### PR TITLE
Minor updates: Spelling and Grammar checks

### PR DESCRIPTION
**Pull request by ragashreeshekar**
Thanks for this fantastic work. 
I was going through the paper and found it could use a couple of spell/grammar updates, hence the PR. 
Please let me know the feasibility. Thanks

### CODE REVIEW

File: whitepapers/serverless-overview/README.md [hunk 1]
```diff
@@ -70,15 +70,15 @@ While serverless computing is widely available, it is still relatively new. In g
...
+For example, for common HTTP-based applications, there are clear upsides in terms of automated scale and a finer-grained cost model. That said, there can be some trade-offs in using a serverless platform. For example, function start-up after a period of inactivity may result in performance declines if the number of instances of the function drops down to zero. Therefore, choosing whether to adopt a serverless architecture requires a careful look at both the functional and non-functional aspects of the compute model.
 
 Non-HTTP-centric and non-elastic scale workloads that weren’t good fits for an IaaS, PaaS, or CaaS solution can now take advantage of the on-demand nature and efficient cost model of a serverless architecture. Some of these workloads include:
 
 * Executing logic in response to database changes (insert, update, trigger, delete)
 
 * Performing analytics on IoT sensor input messages (such as MQTT messages)
 
-* Handling stream processing (analyzing or modifying data in motion)
+* Handling stream processing (analysing or modifying data in motion)
```
Comment (duglin) [inline whitepapers/serverless-overview/README.md:81]: personally I prefer the "z" version

File: whitepapers/serverless-overview/README.md [hunk 2]
```diff
@@ -834,11 +834,11 @@ In the serverless domain, use cases fall into one of the following categories:
...
 A user needs a way to specify their serverless use case or workflow. For example, one use case could be "do face recognition on a photo when a photo is uploaded onto the cloud storage (photo storage event happens)." Another IoT use case could be “do motion analysis” when a motion detection event is received, then depending on the result of the analysis function, either “trigger the house alarm plus call to the police department” or just “send the motion image to the house owner.” Refer to the use cases section for more detailed information.
 
-AWS provides "step function" primitives (state machine based primitives) for the user to specify its workflow, but Step Functions does not allow specification of what event/events will trigger what functions in the workflow. Please refer to [https://aws.amazon.com/step-functions/](https://aws.amazon.com/step-functions/). 
+AWS provides "step function" primitives (state machine-based primitives) for the user to specify its workflow, but Step Functions does not allow specification of what event/events will trigger what functions in the workflow. Please refer to [https://aws.amazon.com/step-functions/](https://aws.amazon.com/step-functions/).
```
Comment (duglin) [inline whitepapers/serverless-overview/README.md:841]: it's interesting that you prefer using a "-" here but not in "auto-scale" :-)

File: whitepapers/serverless-overview/README.md [hunk 3]
```diff
@@ -862,27 +862,27 @@ States and associated information need to be saved in some persistent storage fo
...
 
-The workflow tool might also provide historic data from all prior executions of the workflow, documenting which branches were taken, which data was referenced for a decision to take a certain branch, and so on. This data can be used as an audit log (for example, in industries with strict regulatory requirements) or to analyze and improve performance of the multi-function workflow.
+The workflow tool might also provide historic data from all prior executions of the workflow, documenting which branches were taken, which data was referenced for a decision to take a certain branch, and so on. This data can be used as an audit log (for example, in industries with strict regulatory requirements) or to analyse and improve performance of the multi-function workflow.
```
Comment (duglin) [inline whitepapers/serverless-overview/README.md:865]: z?

**Comment by duglin** [comment]
Overall, I'm ok with this. Just my personal preference for 'z' over 's' :-)
Will add this to the agenda for next week's WG call.
thanks!

**Review by tsurdilo — APPROVED** [review]

File: whitepapers/serverless-overview/README.md (hunk 1 above)
Comment (loopingz) [reply whitepapers/serverless-overview/README.md:81]: Agreed with @duglin we should keep the US version

**Comment by duglin** [comment]
@ragashreeshekar thanks for this - we discussed this on our call last week and most people seemed to prefer if we stuck with us-english instead of uk-english. Could you adjust the few spots I pointed out?

**Pull request merged**